### PR TITLE
Fix switch + Main bugs

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: src/main/scala/cromwell/engine/package.scala [hunk 1]
```diff
@@ -30,6 +30,8 @@ import scala.util.{Failure, Success, Try}
  * Internally, this package is built on top of [[cromwell.binding]].
  */
 package object engine {
+  private val defaultJobAvoidanceValue = false
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/package.scala]: should this be capitalized?

**Comment by scottfrazer** [comment]
Should we also change the terminology from "job avoidance" to call caching?

**Comment by scottfrazer** [comment]
:+1:

File: src/main/scala/cromwell/engine/package.scala [hunk 2]
```diff
@@ -30,6 +30,8 @@ import scala.util.{Failure, Success, Try}
  * Internally, this package is built on top of [[cromwell.binding]].
  */
 package object engine {
+  private val DEFAULT_CALL_CACHING_VALUE = false
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/package.scala]: oh whoops I meant `DefaultCallCachingValue`, all caps with underscores is tres Java.  :smile:

Comment (geoffjentry) [inline src/main/scala/cromwell/engine/package.scala]: zut alors!

**Comment by mcovarr** [comment]
:+1: delta a little less capitalization :smile:

File: src/main/scala/cromwell/engine/package.scala (hunk 2 above)
Comment (Horneth) [inline src/main/scala/cromwell/engine/package.scala]: My old Java demons probably woke up when you said capitalized. Damn it I knew this "Java exorcist" on craigslist was a scam...

**Pull request merged**